sketch_map_tool/routes.py

```diff
@@ -165,7 +167,11 @@ def create_results_post(lang="en") -> Response:
     task_sketch_map = tasks.generate_sketch_map.apply_async(
         args=(bbox, format_, orientation, size, scale, layer, aruco)
     )
-    return redirect(url_for("create_results_get", lang=lang, uuid=task_sketch_map.id))
+    return redirect(
+        url_for(
+            "create_results_get", lang=lang, uuid=task_sketch_map.id, bbox=bbox_wgs84
+        )
```

Comment:
```suggestion
        url_for(
            "create_results_get",
            lang=lang,
            uuid=task_sketch_map.id,
            bbox=bbox_wgs84,
        )
```

I like multilane parameters more because otherwise it can look like 
![Uploading image.png…]()

Just a personal preference..

Reply:
I used `ruff format`. Doesn't this interfere with the strict formating `ruff` does?